appcompat-theme/build.gradle

```diff
@@ -45,7 +43,7 @@ android {
     }
 
     composeOptions {
-        kotlinCompilerExtensionVersion Libs.AndroidX.Compose.version
+        kotlinCompilerExtensionVersion libs.versions.compose.get()
     }
```

Comment:
Is the `composeOptions` block still necessary? AFAIK it was deprecated and is no longer needed but I might be missing something

Reply:
It's still necessary to set the current `kotlinCompilerExtensionVersion` to match your dependency, otherwise you'll get whatever the default is from the AGP version you're using (i.e. will nearly always be older than what you're using).

You might be thinking about the [`kotlinCompilerVersion`](https://developer.android.com/reference/tools/gradle-api/4.1/com/android/build/api/dsl/ComposeOptions#kotlincompilerversion)? You no longer need to set that, as it now just uses whatever Kotlin version you're using.